pootle/apps/virtualfolder/migrations/0004_cleanup_vfolder_names.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def rename(vfolder):
+    i = 0
+    old_name = vfolder.name
+    while True:
```

Comment:
i actually hate this pattern anyway - and its worth mentioning that this commit may die - in favour of addressing https://github.com/translate/pootle/issues/5217